op-e2e/system_adminrpc_test.go

```diff
@@ -217,6 +219,14 @@ func TestPostUnsafePayload(t *testing.T) {
 func assertPersistedSequencerState(t *testing.T, stateFile string, expected node.RunningState) {
 	configReader := node.NewConfigPersistence(stateFile)
 	state, err := configReader.SequencerState()
-	require.NoError(t, err)
+	require.NoError(t, err, "failed to read sequencer state from %s: %v", stateFile, err)
 	require.Equalf(t, expected, state, "expected sequencer state %v but was %v", expected, state)
 }
+
+func TestCleanup(t *testing.T) {
```

Comment:
This is not needed. `t.TempDir` cleans up the directory when the test is done, and even if it didn't, `t.TempDir` creates a new directory each time so this `TestCleanup` function would not cleanup the directory created in the other tests.